### ISSUE TITLE
Update helm charts to use new event-service

### ISSUE THREAD

**Issue by abbi-gaurav** [kyma-project/kyma#405]
Note:
 * This task will need the documentation to be present for the PR to be raised. (Note: the documentation will be done separately here #331)
 * The code clean up and helm chart changes can be done in parallel with documentation task.

Hint:

* Use the local workshop steps for verification

AC:
 * Remove code for eventing from `kyma/components/gateway`
 * Create new binary for `kyma/components/gateway`
 * Create new helm charts for `kyma/components/event-service`
 * Update helm charts for `kyma/components/gateway` to route event publishing to event-service service
 * e2e-tests should succeed
 * Gateway functionality works as expected (metadata, event API and proxy)
 * Clean up the [value.yaml](https://github.com/kyma-project/kyma/blob/master/resources/remote-environments/values.yaml) by remove unneeded configs.

**Comment by marcobebway** [comment]
Product documentation for the `Event-Service` will be done in a separate issue #331.

**Comment by marcobebway** [comment]
These are the PRs related to this issue:
- #182
- #166
- #333
- #586
